src/doom/p_mobj.c

```diff
@@ -727,16 +727,16 @@ void P_RespawnSpecials (void)
     int			i;
 
     // only respawn items in deathmatch
-    if (deathmatch != 2)
-	return;	// 
+    if (deathmatch != 2 && deathmatch != 3)
+	return;
```

Comment:
Please revert the changes to the three `return;` lines. We do not apply cosmetic changes to Choco code in line that we don't change otherwise.